starcounter-include.html

```diff
@@ -316,13 +310,55 @@
         * @param  {Mixed} value New value
         */
         StarcounterIncludePrototype._setPendingPropertyOrPath = function (path, value) {
-            var sameModelAlreadyLoaded = this.template.model == this.partial;
-            this.partialChanged(this.partial);
-            if (sameModelAlreadyLoaded && this.template._setPendingPropertyOrPath) {
-                this.template._setPendingPropertyOrPath(
-                    path.replace("partial.", "model.")
-                        .replace("viewModel.", "model."),
-                     value);
+
+            var sameModelAlreadyLoaded = this.template.model === this.partial;
+            const newPath = path.replace("partial.", "model.")
+                .replace("viewModel.", "model.");
+            // If that's the same model, we still may need to bump
+            if(sameModelAlreadyLoaded){
+                // yey we still support both names, lets strip one or another.
+                const subPath = (path.indexOf('partial.') === 0) && path.substr(8) ||
+                                (path.indexOf('viewModel.') === 0) && path.substr(10);
```

Comment:
`path.beginsWith('partial.')` would be easier to read

Reply:
I even thought about it, but lack of cross-browser support made me do it that way.  But you are right, that's why we dropped IE support to make our life easier.

Reply:
For `starcounter-include.html`, IE does not exist.